src/equations/shallow_water_1d.jl

```diff
@@ -547,6 +582,30 @@ end
     return waterheight(u, equations) * pressure(u, equations)
 end
 
+"""
+    calc_wavespeed_roe(u_ll, u_rr, direction::Integer,
+                       equations::ShallowWaterEquations1D)
+
+Calculate Roe-averaged velocity `v_roe` and wavespeed `c_roe = sqrt{g h_roe}`
```

Comment:
Can you link a reference for this?

Reply:
Addressed in ef6d070a63118991967d19ebcfa1fa1a953b41ee